buildSrc/build.gradle.kts

```diff
@@ -17,3 +17,16 @@ repositories {
     google()
     mavenCentral()
 }
+
+java {
+    // Use Java 8 instead of Java 11
+    // See https://github.com/gradle/gradle/issues/18935
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
+dependencies {
+    implementation("com.android.tools.build:gradle:7.2.0")
+    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.6.21") //https://issuetracker.google.com/issues/176079157#comment14
+    implementation("com.squareup:javapoet:1.13.0") // https://github.com/google/dagger/issues/3282
```

Comment:
I don't know what is better but you have a comment with a space after `//` and the other not 🤓